examples/net6.0/aspnetcore/Controllers/RedisController.cs

```diff
@@ -22,20 +22,22 @@ public RedisController(ILogger<RedisController> logger, IDatabase database)
         _redisDb = database;
     }
 
-    [HttpPost]
-    public async Task<ActionResult<string>> LeftPush([FromBody] LeftPushBody data)
+    [HttpGet]
+    // public async Task<ActionResult<string>> LeftPush([FromBody] LeftPushBody data)
+    public async Task<ActionResult<string>> LeftPush()
```

Comment:
The changes in this file should also go into _examples/**net8.0**/aspnetcore/Controllers/RedisController.cs_.